z3/tests/lib.rs

```diff
@@ -829,3 +829,150 @@ fn test_issue_94() {
     let i1 = ast::Int::fresh_const(&ctx1, "b");
     ast::Int::add(&ctx0, &[&i0, &i1]);
 }
+
+#[test]
+/// Given COUNT items with indices 0..COUNT,
+/// find a function that reverses the indices with soft assertions.
+///
+/// An additional, incorrect soft assertion (COUNT-1 > 0) is used to
+/// show that optimization incurred the least penalty.
```

Comment:
Can you put this in a new `objectives.rs` test file or something like that? Thanks.

Reply:
sure thing